scripts/prettyjson

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+# Reformat all JSONs in the source tree
+
+for basedir in inspirehep/modules/records/jsonschemas/ \
+        inspirehep/modules/workflows/mappings/ \
+        inspirehep/modules/records/mappings/; do
+    for name in `find $basedir -name *.json`; do
+        echo Cleaning $name
+        python -c "import json; input = json.load(open('${name}')); output = json.dumps(input, indent=4, sort_keys=True); open('${name}', 'w').write(output.replace(', \n', ',\n') + '\n')"
```

Comment:
This can be just `python -m json.tool`:
```shell
% echo '{"foo": "foo", "bar": "baz"}' | python -m json.tool
{
    "bar": "baz",
    "foo": "foo"
}
```